### PR TITLE
Iss1

### PR DESCRIPTION
**Pull request by San411**
1) train_cnn.ipynb - The os.system method throws an error. I've replaced it with os.symlink() to prevent the ln command. The symlink creation can be verified.(Commands in the readme file).

2) download_images.ipynb - While dropping the images that are already downloaded, it happens that some images present in the csv file ' image_download_locs.csv ' are not present in the countries/<country_name> directory and vice versa. So i have included only those images that are common in both the files.

Do let me know what you think of it.

### CODE REVIEW

**Comment by jmathur25** [comment]
Hey, I don't have too much time to look at this but generally what you have seems fine. Few important things:

1. "While dropping the images that are already downloaded, it happens that some images present in the csv file ' image_download_locs.csv ' are not present in the countries/<country_name> directory and vice versa." Are you sure? The logic of dropping images that were not downloaded seems correct.

2. I won't merge the PR directly because your changes include more than just the minimum (there is your history + some other little things changed here and there). Can you summarize what you want to change in master so I can just apply that in one commit? The README diff is pretty clear, but with Jupyter Notebooks the diff can be a little troublesome to read.

**Comment by San411** [comment]
This fork has a lot of commit history in the main branch I'm sorry about that. I will delete this fork and raise another PR from a new one with just the essential commits. I'm closing this.
